src/router/handlers/resourceHandler.test.ts

```diff
@@ -1061,3 +1062,38 @@ describe('Testing history', () => {
         });
     });
 });
+
+describe('Testing xhtml validation', () => {
+    test('valid patient resource is not affected', () => {
+        if (process.env.VALIDATE_XHTML !== 'true') {
```

Comment:
I think for tests we should be validating resources whether or not `process.env.VALIDATE_XHTML === 'true'`

Reply:
Good point, updated! I also realized that the utility function itself was also checking the env var in addition to the place it's actually used, so I updated that to be consistent with the tests as well.